pylib/anki/consts.py

```diff
@@ -16,6 +16,7 @@
 
 # Card and queue types
 QUEUE_TYPE_NEW = 0
+QUEUE_TYPE_LRN = 1
```

Comment:
type and queue are related, but not interchangeable. For example, type=3 means a relearning card, but queue=3 means a learning or relearning card with a 24+ hour interval. For this reason, there should be separate constants for TYPE and QUEUE, as can be seen on the work the Agenty did already:  

CARD_TYPE_RELEARNING = 3
QUEUE_TYPE_PREVIEW = 4

Reply:
QUEUE and CARD are separated.
I should emphasize that this is really not a clear name. I used this constant name to follow constants already introduced in the code. However, it's strange that both CARD_TYPE_RELEARNING and QUEUE_TYPE_PREVIEW contain the word type, while only the former is related to the attribute "type" in the database/card object. QUEUE_PREVIEW would makes more sens I believe, in order to avoid confusion